### PR TITLE
Added Service Redirect Backup Policy and L1/L2 Destination resources

### PR DESCRIPTION
**Pull request by sajagana**
Note:

Added Service Redirect Backup Policy and L1/L2 Destination resources
Created a common example file for Service Redirect Policy and Service Redirect Backup Policy

### CODE REVIEW

File: website/docs/r/pbr_l1_l2_destination.html.markdown [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+## Argument Reference ##
+
+* `policy_based_redirect_dn` - (Required) Distinguished name of the parent Policy-Based Redirect object.
+* `dest_name` - (Required) Destination Name of the L1/L2 redirected traffic.
+* `annotation` - (Optional) Annotation of the L1/L2 redirected traffic.
+* `name_alias` - (Optional) Name Alias of the L1/L2 redirected traffic.
+* `name` - (Optional) Name of the L1/L2 redirected traffic.
+* `mac` - (Optional) MAC Address of the L1/L2 redirected traffic.
+* `pod_id` - (Optional) Pod Id of the L1/L2 redirected traffic. Allowed range is 1-255 and default value is "1".
+* `relation_vns_rs_l1_l2_redirect_health_group` - (Optional) Represents the relation to a L4-L7 Redirect Health Group (class vnsRedirectHealthGroup). Type: String.
```
Comment (akinross) [inline website/docs/r/pbr_l1_l2_destination.html.markdown:63]: When argument is of type string is "Type: String" needed in the text? Applies for the other documentation also in this PR.

File: website/docs/r/pbr_l1_l2_destination.html.markdown [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  relation_vns_rs_to_c_if                     = aci_concrete_interface.concrete_interface.id
+  relation_vns_rs_l1_l2_redirect_health_group = aci_l4_l7_redirect_health_group.l4_l7_health_group.id
+}
+
+```
+
+## Argument Reference ##
+
+* `policy_based_redirect_dn` - (Required) Distinguished name of the parent Policy-Based Redirect object.
+* `dest_name` - (Required) Destination Name of the L1/L2 redirected traffic.
```
Comment (akinross) [inline website/docs/r/pbr_l1_l2_destination.html.markdown:57]: Do we need object(s) appended to L1/L2 redirected traffic? And capitalise R and T?

File: website/docs/r/service_redirect_backup_policy.html.markdown [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+* `tenant_dn` - (Required) Distinguished name of the parent Tenant object.
+* `name` - (Required) Name of the PBR Backup Policy object.
+* `annotation` - (Optional) Annotation of the PBR Backup Policy object.
+* `name_alias` - (Optional) Name Alias of the PBR Backup Policy object.
+
+
+## Importing ##
+
+An existing PBRBackupPolicy can be [imported][docs-import] into this resource via its Dn, via the following command:
```
Comment (anvitha-jain) [inline website/docs/r/service_redirect_backup_policy.html.markdown:43]: PBRBackupPolicy -> PBR Backup Policy

**Review by anvitha-jain — CHANGES_REQUESTED** [review]

File: website/docs/d/pbr_l1_l2_destination.html.markdown [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+```
+
+## Argument Reference ##
+
+* `policy_based_redirect_dn` - (Required) Distinguished name of the parent Policy-Based Redirect object.
+* `dest_name` - (Required) Destination Name of the L1/L2 Redirected Traffic.
+
+## Attribute Reference ##
+* `id` - Attribute id set to the Dn of the L1/L2 Redirected Traffic.
```
Comment (lhercot) [inline website/docs/d/pbr_l1_l2_destination.html.markdown:50]: of the destination of the L1/L2 Redirected Traffic.

File: website/docs/d/pbr_l1_l2_destination.html.markdown [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+* `annotation` - (Optional) Annotation of the L1/L2 Redirected Traffic.
```
Comment (lhercot) [inline website/docs/d/pbr_l1_l2_destination.html.markdown:51]: of the destination of the L1/L2 Redirected Traffic.

File: website/docs/d/pbr_l1_l2_destination.html.markdown [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+* `name_alias` - (Optional) Name Alias of the L1/L2 Redirected Traffic.
```
Comment (lhercot) [inline website/docs/d/pbr_l1_l2_destination.html.markdown:52]: of the destination of the L1/L2 Redirected Traffic.

File: website/docs/d/pbr_l1_l2_destination.html.markdown [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+* `name` - (Optional) Name of the L1/L2 Redirected Traffic.
```
Comment (lhercot) [inline website/docs/d/pbr_l1_l2_destination.html.markdown:53]: of the destination of the L1/L2 Redirected Traffic.

File: website/docs/d/pbr_l1_l2_destination.html.markdown [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+* `mac` - (Optional) MAC Address of the L1/L2 Redirected Traffic.
```
Comment (lhercot) [inline website/docs/d/pbr_l1_l2_destination.html.markdown:54]: of the destination of the L1/L2 Redirected Traffic.

File: website/docs/d/pbr_l1_l2_destination.html.markdown [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+* `pod_id` - (Optional) Pod Id of the L1/L2 Redirected Traffic.
```
Comment (lhercot) [inline website/docs/d/pbr_l1_l2_destination.html.markdown:55]: of the destination of the L1/L2 Redirected Traffic.

File: website/docs/d/pbr_l1_l2_destination.html.markdown [hunk 10]
```diff
@@ -0,0 +1,57 @@
...
+data "aci_pbr_l1_l2_destination" "l1_l2_destination" {
+  policy_based_redirect_dn = aci_service_redirect_backup_policy.pbr_backup.id
+  dest_name                = "l1_l2_destination"
+}
+
+```
+
+## Argument Reference ##
+
+* `policy_based_redirect_dn` - (Required) Distinguished name of the parent Policy-Based Redirect object.
```
Comment (lhercot) [inline website/docs/d/pbr_l1_l2_destination.html.markdown:46]: Distinguished name of the parent Policy-Based (Redirect or Redirect Backup) object

File: website/docs/r/pbr_l1_l2_destination.html.markdown [hunk 11]
```diff
@@ -0,0 +1,75 @@
...
+  relation_vns_rs_to_c_if                     = aci_concrete_interface.concrete_interface.id
+  relation_vns_rs_l1_l2_redirect_health_group = aci_l4_l7_redirect_health_group.l4_l7_health_group.id
+}
+
+```
+
+## Argument Reference ##
+
+* `policy_based_redirect_dn` - (Required) Distinguished name of the parent Policy-Based (Redirect or Redirect Backup) object.
+* `dest_name` - (Required) Destination Name of the L1/L2 Redirected Traffic.
```
Comment (lhercot) [inline website/docs/r/pbr_l1_l2_destination.html.markdown:57]: of the destination of the L1/L2 Redirected Traffic.

File: website/docs/r/pbr_l1_l2_destination.html.markdown [hunk 12]
```diff
@@ -0,0 +1,75 @@
...
+* `annotation` - (Optional) Annotation of the L1/L2 Redirected Traffic.
```
Comment (lhercot) [inline website/docs/r/pbr_l1_l2_destination.html.markdown:58]: of the destination of the L1/L2 Redirected Traffic.

File: website/docs/r/pbr_l1_l2_destination.html.markdown [hunk 13]
```diff
@@ -0,0 +1,75 @@
...
+* `name_alias` - (Optional) Name Alias of the L1/L2 Redirected Traffic.
```
Comment (lhercot) [inline website/docs/r/pbr_l1_l2_destination.html.markdown:59]: of the destination of the L1/L2 Redirected Traffic.

File: website/docs/r/pbr_l1_l2_destination.html.markdown [hunk 14]
```diff
@@ -0,0 +1,75 @@
...
+* `name` - (Optional) Name of the L1/L2 Redirected Traffic.
```
Comment (lhercot) [inline website/docs/r/pbr_l1_l2_destination.html.markdown:60]: of the destination of the L1/L2 Redirected Traffic.

File: website/docs/r/pbr_l1_l2_destination.html.markdown [hunk 15]
```diff
@@ -0,0 +1,75 @@
...
+* `mac` - (Optional) MAC Address of the L1/L2 Redirected Traffic.
```
Comment (lhercot) [inline website/docs/r/pbr_l1_l2_destination.html.markdown:61]: of the destination of the L1/L2 Redirected Traffic.

File: website/docs/r/pbr_l1_l2_destination.html.markdown [hunk 16]
```diff
@@ -0,0 +1,75 @@
...
+* `pod_id` - (Optional) Pod Id of the L1/L2 Redirected Traffic. Allowed range is 1-255 and default value is "1".
```
Comment (lhercot) [inline website/docs/r/pbr_l1_l2_destination.html.markdown:62]: of the destination of the L1/L2 Redirected Traffic.

File: website/docs/r/pbr_l1_l2_destination.html.markdown [hunk 17]
```diff
@@ -0,0 +1,75 @@
...
+* `relation_vns_rs_l1_l2_redirect_health_group` - (Optional) Represents the relation to a L4-L7 Redirect Health Group (class vnsRedirectHealthGroup).
+* `relation_vns_rs_to_c_if` - (Optional) Represents the relation to a Concrete Interface (class vnsCIf).
+
+
+## Importing ##
+
+An existing L1/L2 Redirected Traffic can be [imported][docs-import] into this resource via its Dn, via the following command:
```
Comment (lhercot) [inline website/docs/r/pbr_l1_l2_destination.html.markdown:69]: An existing Destination of the L1/L2 Redirected Traffic...

**Review by lhercot — CHANGES_REQUESTED** [review]

**Review by lhercot — APPROVED** [review]
LGTM

**Review by akinross — APPROVED** [review]
LGTM

**Pull request merged**